### PR TITLE
11.0 mig website form metada

### PR DESCRIPTION
**Pull request by chienandalu**
Website Form Metadata
=====================

Odoo has the hidden and unused ability to gather metadata from the website
forms and record into the record created from such form. This module unveils
it.

cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Get the occasion to use new README system

**Review by rafaelbn — APPROVED** [review]

**Review by yajo — APPROVED** [review]
Functional & code review

**Comment by chienandalu** [comment]
@pedrobaeza Ok, let's try it

File: website_form_metadata/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[ This file is optional, it should explain how to configure
```
Comment (pedrobaeza) [inline website_form_metadata/readme/CONFIGURE.rst]: You don't need to include this explanatory texts.

Comment (chienandalu) [reply website_form_metadata/readme/CONFIGURE.rst]: Ok, is it needed to configure some bot in the repo?

Comment (pedrobaeza) [reply website_form_metadata/readme/CONFIGURE.rst]: No, this is done inside OCA server all nights in a process that scans all OCA repositories and update the needed things.

Comment (chienandalu) [reply website_form_metadata/readme/CONFIGURE.rst]: Ok, done

**Comment by yajo** [comment]
@chienandalu is this ready to merge?

**Comment by chienandalu** [comment]
I'd say it is. @pedrobaeza ?

**Comment by pedrobaeza** [comment]
Is Odoo PR merged?

**Comment by chienandalu** [comment]
It was forward-ported from the one in `10.0`: https://github.com/odoo/odoo/commit/373a1c1128cd972630a26d4063e153ef6f7248fb

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**